src/udiskslinuxfilesystem.c

```diff
@@ -183,6 +186,11 @@ udisks_linux_filesystem_update (UDisksLinuxFilesystem  *filesystem,
   dev = udisks_linux_block_object_get_device_file (object);
   type = g_udev_device_get_property (device->udev_device, "ID_FS_TYPE");
   size = 0;
+
+  /* skip get_info when drive is inactive */
+  if (!udisks_linux_drive_ata_is_active(device, NULL, &state))
```

Comment:
Code style - please add space before the opening parenthesis.

Also, `state` is undefined.

The case of non-ATA capable devices is still not handled properly.